src/lv_gpu/lv_gpu_stm32_dma2d.c

```diff
@@ -11,19 +11,14 @@
 
 #if LV_USE_GPU_STM32_DMA2D
 
-#if defined(STM32F4)
-    #include "stm32f4xx_hal.h"
-#elif defined(STM32F7)
-    #include "stm32f7xx_hal.h"
-#else
-    #error "Not supported STM32 family to use DMA2D"
-#endif
+#include "stm32f429xx.h"
```

Comment:
Can't we simply replace the includes above with `stm32f7xx.h` and `stm32f4xx.h`?

Reply:
Problem is STM32F4 and STM32F7 aren't defined unless you are using the CubeMX HAL